### ISSUE TITLE
Removing jQuery dependency

### ISSUE THREAD

**Issue by JohnJVTK** [wenzhixin/multiple-select#570]
Hello,

Do you have any plans on removing jQuery dependency ?

Thanks,
John

**Comment by ghiscoding** [comment]
I'm currently working on a new forked repo for a pure/vanilla JS lib, it's still under development but I already have a live demo and I'm expecting to publish a first release in the coming days. The reason I created this new repo is because I use this great lib for some other Open Source libs that I have and the lib seems to be unmaintained since a while and some users might want to keep the jQuery version anyway since the vanilla one will be slightly bigger because it requires couple of DOM utils functions. I could also transfer the lib back to its owner but I wish to keep it maintained at the same time.

in short, take a look at this new repo [multiple-select-vanilla](https://github.com/ghiscoding/multiple-select-vanilla), there's a [live demo](https://ghiscoding.github.io/multiple-select-vanilla/) and I'm expecting to publish a version in coming days. I will also add extra features like auto-position (top/bottom & left/right), I already have the code in another fork

**Comment by JohnJVTK** [comment]
@ghiscoding 
Thanks !
In the meantime I switched to davidstutz plugin but I might give a try to your fork.

**Comment by ghiscoding** [comment]
I did a few releases since then and I'm currently at `0.4.0` and I am now using it in production. I also updated the `v0.4.0` to align with this `v1.6.0`, so both projects should still be in sync.

> In the meantime I switched to davidstutz plugin but I might give a try to your fork.

I'm not sure why you mentioned this project since it's another jQuery lib, so it seems unrelated to this opened issue (which was to remove jQuery lol)
